### ISSUE TITLE
Reintroduce excludeSubpageTypes for v:page.menu.browse

### ISSUE THREAD

**Issue by SadlerJohnson** [FluidTYPO3/vhs#1329]
We include a browse menu for each section of a 6000 page website. Around 250 of these pages are shortcuts for alternative spellings or synonyms. These shortcuts can redirect to the next article or an article in another branch of the page tree altogether. The issue is, that hitting these shortcuts via next and previous can take the user out of the current section or, worse, keep them in a loop within the section. The worst case scenario is when the synonym is directly after its respective main article.
I could exclude these 250 pages with excludePages but that would be cumbersome at best and requires editors to change the template when adding shortcuts. Therefore, I request that consideration is given to reintroducing excludeSubpageTypes which was deprecated after VHS 2.4.0.

**Comment by NamelessCoder** [comment]
Not an unreasonable request given the use case. @bjo3rnf and @cedricziel what do you guys think? Shall we restore this or are we somehow prevented from that by the APIs we use?

@SadlerJohnson if you need a quick fix for this you could do a bit of manual rendering by pulling a standard menu and combining manual rendering with the `v:iterator.next`, `v:iterator.previous`, first, last and so on ViewHelpers to handle your next/prev linking. This is by the way a trick you can also use for pagination purposes - and I hope it helps in this case too.

**Comment by bjo3rnf** [comment]
Agreed. Requires some refactoring though but I'll put it on the todo list.
